content/en/docs/concepts/storage/storage-limits.md

```diff
@@ -67,10 +67,9 @@ type and enforces the appropriate maximum number of volumes for the node. For ex
 up to 127 volumes can be attached to a node, [depending on the node
 type](https://cloud.google.com/compute/docs/disks/#pdnumberlimits).
 
-* For Amazon EBS disks on M5,C5,R5,T3 and Z1D instance types, Kubernetes allows only 25
-volumes to be attached to a Node. For other instance types on
+* On
 <a href="https://aws.amazon.com/ec2/">Amazon Elastic Compute Cloud (EC2)</a>,
-Kubernetes allows 39 volumes to be attached to a Node.
+up to 127 volumes can be attached to a node, [depending on the node's underlying instance type](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/volume_limits.html).
```

Comment:
```suggestion
up to 127 volumes can be attached to a Node, [depending on the node's underlying instance type](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/volume_limits.html).
```
Deploy preview looks good! Just a small nit to capitalize Node since it's an API type. Good to go after that.

Reply:
Thanks! Updated capitalized Node in the other bullets too for consistency.